src/Assignments/SimpleProblemPage.tsx

```diff
@@ -70,8 +70,20 @@ export const SimpleProblemPage: React.FC<SimpleProblemPageProps> = () => {
 
         const currentTopic = res.data.data.topic;
 
-        if (currentTopic.studentTopicOverride?.length > 0) {
-            _.assign(currentTopic, currentTopic.studentTopicOverride[0]);
+        // apply topic overrides: start/end/dead dates
+        if (!_.isNil(currentTopic.studentTopicOverride) && !_.isEmpty(currentTopic.studentTopicOverride)) {
+            const override = currentTopic.studentTopicOverride[0];
+            delete override.id;
```

Comment:
This is not a good way to do it
In some cases there are foreign keys that are in common as well
We should keep this consistent and have an interface for the data portion of the overrides table and then do the assign based on a pick

Reply:
Ahh I thought this was the backend for a second, I guess we never really did it right on the frontend